src/webapp/src/characters/Characters.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { Button } from '@small-ads/ui';
```

Comment:
I think changes in this file are a bit irrelevant on the scope of this pr. Would be cool if you take a moment and revert them is good practise to focus on one thing at a time on prs helps both you and the reviewers have a focused scope 😎